docs/glossary.md

```diff
@@ -0,0 +1,202 @@
+---
```

Comment:
Some that I think are missing:
- granule
- CNM
- CMR
- DAAC

Reply:
👍  on the last three.        Granule could be a pretty complex definition 🤔

Reply:
ffc8d38d, 7ea49dea.   We don't yet have a link to the CNM standard.